QuasistaticBrownianThermalNoise/config.yaml

```diff
@@ -22,19 +22,20 @@ Mirror:
   # The prefix Iso_ means isotropic.
   Materials:
     SubstrateMaterial: Iso_FusedSilica
-    CoatingMaterial: AlGaAs
+    CoatingMaterial: Iso_AlGaAs
 
   # Mirror temperature in Kelvin, for computing the thermal noise.
   Temperature: 300.0
 
 Resolution:
   # Refine the mesh each iteration, for Cycles iterations total. Higher
   # Cycles means run to higher resolution.
-  Cycles: 1
+  Cycles: 9
```

Comment:
Please leave this as 1, so by default the tests run fast. Add a comment that you should set higher (e.g. 9 or higher) for using multiple loss angles. I say this because 9 is probably too much to run on a laptop. Or do you have a good reason for 9 here?

Reply:
I assumed that we would only be running on clusters, not on laptops. I will change it back to 1.